docs/source/posts/2024/2024-08-06-week6-wachiou-bouraima.rst

```diff
@@ -0,0 +1,92 @@
+WEEK 6: Code reviews, relining and crush challenges
+===================================================
+
+.. post:: August 6, 2024
+   :author: Wachiou BOURAIMA
+   :tags: google
+   :category: gsoc
+
+Hello everyone,
+As my Google Summer of Code (GSoC) 2024 journey progresses, week6 has brought me a series of technical challenges and accomplishments. My main focus has been on code reviews, rebasing and commits squasing, with a few notable lessons learned along the way.
```

Comment:
`squasing`  ->  `squashing`

Reply:
Hi @itellaetxe !

Thank you for your comments